apps/src/p5lab/drawUtils.js

```diff
@@ -22,6 +22,39 @@ export function getTextWidth(p5, text, size) {
   return width;
 }
 
+/**
+ * Draws a bubble on the canvas that displays a piece of text, representing a variable name and value.
+ * The bubble has a black background with a white border and white text.
+ * The bubble is drawn with rounded corners. The text is horizontally and vertically centered within the bubble.
+ *
+ * The bubble's position is based on the x and y parameters which represent the center of the bubble.
+ * The size of the bubble is dynamically calculated based on the width of the text and padding.
+ *
+ * @param {p5} p5 - The p5 instance used to draw the bubble.
+ * @param {number} x - The x-coordinate of the center of the bubble.
+ * @param {number} y - The y-coordinate of the center of the bubble.
+ * @param {string} text - The text to display inside the bubble. This should represent the variable's name and value.
+ */
+export function variableBubble(p5, x, y, text) {
+  const padding = 15;
+  const textSizeValue = 35; // Text size matches the text size used for title page subtitles
```

Comment:
This feels too big to me. Not a blocker since it's easy to iterate on. FWIW, we use size 20 for speech bubbles.